### ISSUE TITLE
[FEATURE] faster processing of LLM queries

### ISSUE THREAD

**Issue by DefJM** [DefJM/openOwl#3]
## TLDR
- Problem: LLM queries for toxicity classification take too long. Older libraries have 100s of issues and result in thousand+ comments to classify for toxicity. Current implementation is buggy (stops and has to be manually fixed) and slow (takes hours)
- Suggested solution: Implement asynchronous API calls. 

### Researching asynchronous API calls for different LLMs
- Anthropic:
  - Message Batches https://docs.anthropic.com/en/docs/build-with-claude/message-batches
- Ollama:
  - https://www.reddit.com/r/ollama/comments/1dhqm7i/best_way_to_use_concurrency_and_workload/
- OpenAI:
  -  Batch API https://platform.openai.com/docs/guides/batch

### Alternative approach
- Try different prompt strategy asking LLM to label batch of comments. (We think this will deteriorate classification quality, as the LLM's context is messed up through multiple comments.)
